packages/headless/src/app/logger-middlewares.test.ts

```diff
@@ -1,63 +1,92 @@
 import {SchemaValidationError} from '@coveo/bueno';
```

Comment:
Changed from testing through the engine itself + `redux-mock-store`, to testing the middleware directly, using this approach: https://redux.js.org/usage/writing-tests#middleware